pubspec.yaml

```diff
@@ -4,7 +4,7 @@ description: The setup-dart GitHub Action - download and setup the Dart SDK.
 publish_to: none
 
 environment:
-  sdk: ^2.19.0
+  sdk: ^3.0.0-0
```

Comment:
I'd use a more recent 3.0.0 version, since this might version resolve to an older 3.0.0 version that doesn't have the needed interop lowerings (or even inline class support). Maybe the latest beta release?

Reply:
Good point, I went with `3.0.0-356` as that is the one we know for sure works from CI